### DISCUSSION TITLE
Welcome to Ollama Discussions!

### DISCUSSION THREAD

**Discussion by HavenDV** [tryAGI/Ollama#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MiaJohnson** [comment]
I can't create a bug, I can't post a discussion, I can't interact with any of the normal channels to report issues.
Your tutorial/how-to steps are broken.
Error (active)	CS1729	'OllamaApiClient' does not contain a constructor that takes 0 arguments
